packages/components/src/styles/flyout-dialog-reset-styles.ts

```diff
@@ -0,0 +1,14 @@
+import type { Styles } from 'jss';
+
+export const getFlyoutDialogResetStyles = (): Styles => {
```

Comment:
Wrong type?  I think this is a `JssStyle` and function named should be renamed accordingly.